clients/privacy-center/constants/index.ts

```diff
@@ -1,41 +1,34 @@
 /* eslint-disable import/prefer-default-export */
-// todo- needs to be runtime not buildtime
-import configJson from "~/config/config.json";
+import dangerousStaticDefaultConfig from "~/config/config.json";
```

Comment:
TODO: I still don't love that this import exists, would like to see if it's possible to nuke entirely. It gets used to provide some "default" config options into the reducer, but I think I can potentially remove that...